src/teaching-element/File.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="te-file">
+    <button
+      v-if="url"
+      @click="download"
```

Comment:
Hmm, what is the benefit of doing this dynamic anchor detour instead of rendering it here with prepopulated download and href attributes?